Documentation/ApiOverview/ContentElements/ContentElementsWizard.rst

```diff
@@ -49,23 +47,19 @@ plugin is `registration`.
 
 #. Include TSconfig
 
-   :file:`ext_localconf.php`:
-
    .. code-block:: typoscript
+      :caption: EXT:example/Configuration/page.tsconfig
 
-        \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
-            '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:example/Configuration/TsConfig/Page/Mod/Wizards/NewContentElement.tsconfig">'
-        );
+      @import 'EXT:example/Configuration/TsConfig/Page/Mod/Wizards/NewContentElement.tsconfig'
 
    This always includes the above page TSconfig. It is better practice to make this configurable by
```

Comment:
Let us leave a hint here, that this only works with Version 12 and a link on how to be compatible with version 11 and 12 (There is a seciton in Setting Page TSconfig)